exalearn-ui/src/app/components/test-grammar/test-grammar.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="grammar-module">
+	<div>
+		<button
+			mat-mini-fab
+			color="white"
+			class="nav-arrow-btn"
+			(click)="currentIndex>0 ?currentIndex = currentIndex - 1: currentIndex"
+		>
```

Comment:
Cosmetic: not cool leave this poor little bracket alone on this line

Reply:
I can't do that because in that case git hook or something else doesn't let me commit

Reply:
> Cosmetic: not cool leave this poor little bracket alone on this line

yeah) The git hook made it. Because the git hook automaticaly before committing calls "prettier --write" and then commit changes. I think, it is norm to leave this little poor bracket alone, when we have so much attributes.